src/main/java/org/folio/processing/mapping/mapper/reader/record/marc/MarcRecordReader.java

```diff
@@ -80,14 +82,19 @@ public class MarcRecordReader implements Reader {
   private static final String DATE_TIME_FORMAT = "dd-MM-yyyy HH:mm:ss";
   private static final String UTC_TIMEZONE = "UTC";
   private static final List<String> NEEDS_VALIDATION_BY_ACCEPTED_VALUES = List.of("vendor", "materialSupplier", "accessProvider");
+  private static final String STATISTICAL_CODE_ID_FIELD = "statisticalCodeId";
   private static final String BLANK = "";
 
   private EntityType entityType;
+  private Map<String, AcceptedValuesMatcher> acceptedValuesMatchers;
   private Record marcRecord;
   private MappingParameters mappingParameters;
 
   MarcRecordReader(EntityType entityType) {
     this.entityType = entityType;
+    this.acceptedValuesMatchers = new HashMap<>() {{
```

Comment:
please consider initializing the map via `Map.of()` since the current approach has overhead in object creation under the hood

Reply:
the current approach has been applied because of `Map.of()` implementation doesn't allow a nullable key to be passed into get() method